src/core/dbus-manager.c

```diff
@@ -629,76 +629,42 @@ static int method_reset_failed_unit(sd_bus_message *message, void *userdata, sd_
         return method_generic_unit_operation(message, userdata, error, bus_unit_method_reset_failed);
 }
 
-static int method_set_unit_properties(sd_bus_message *message, void *userdata, sd_bus_error *error) {
-        Manager *m = userdata;
+static int method_generic_unit_operation_loaded_only(sd_bus_message *message, Manager *m, sd_bus_error *error, sd_bus_message_handler_t handler) {
```

Comment:
Maybe, instead of creating new function, how about adding a boolean argument `validate_load_state`?